modules/pel/peltool/peltool.py

```diff
@@ -289,6 +366,14 @@ def main():
                         help='Used with -d, only look for files with this extension (e.g. ".pel")')
     parser.add_argument('-x', '--delete', dest='delete', action='store_true',
                         help='Delete original file after parsing')
+    if inBMC:
+        parser.add_argument('-l', '--list',
+                            help='List PELs',
+                            action='store_true')
+    else:
+        parser.add_argument('-l', '--list',
+                            help='List PELs',
+                            dest='list')
```

Comment:
Suggestion/FYI: I would use https://docs.python.org/3/library/argparse.html#metavar to inform about the directory arguments.

Reply:
Noted!